example/package.json

```diff
@@ -5,7 +5,7 @@
     "clean": "rimraf dist pkg target"
   },
   "devDependencies": {
-    "@wasm-tool/wasm-pack-plugin": "^0.0.5",
+    "@wasm-tool/wasm-pack-plugin": "../",
```

Comment:
I don't think we want this, it means that you can not run the example out of this repo anymore.